README.md

```diff
@@ -18,7 +18,29 @@ To run the tests simply issue:
 
     mvn test
 
-Tests are ran against an in-memory database (H2). To run tests against a MariaDB instance, you can change [`src/test/resources/resources/database.properties`][database.prop] and run the Docker container in the root folder with:
+Note: tests are ran against an in-memory database (H2). 
```

Comment:
```suggestion
Note: tests are run against an in-memory database (H2). 
```

Reply:
I think it was ok "ran", isn't it?

Reply:
> I think it was ok "ran", isn't it?

The correct one is:  Note: tests are run against an in-memory database (H2).

Here's why:

Run vs. Ran: "Run" is the present participle of the verb "to run," which is the correct tense for describing an ongoing action like testing. "Ran" is the past participle, used for completed actions.

"run" is the grammatically correct choice in this context.

@dbmalkovsky what do you think?

Reply:
I think "run" is the correct choice; besides it just sound better to a native English speaker 😏

Reply:
I agree now, so it was wrong before 😆